account_invoice_factur-x/__manifest__.py

```diff
@@ -16,7 +16,7 @@
         'base_zugferd',
         'base_vat',
         ],
-    'external_dependencies': {'python': ['facturx']},
+    'external_dependencies': {'python': ['factur-x']},
```

Comment:
That would be correct but in v13 only, which allows declaring the PyPI distribution name here. Until 12.0, it must be the import name, and the mapping must go in setup.py. You probably need to backport #192